### PR TITLE
Feat: added category page

### PR DESCRIPTION
**Pull request by anjali1102**
image

### CODE REVIEW

**Comment by anjali1102** [comment]
https://quiz-app-mui.netlify.app/

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import Settings from './pages/Settings';
+import FinalScore from './pages/FinalScore';
+import './App.css'; 
+import Questions from './pages/Questions';
```
Comment (jeetsdev) [inline src/App.jsx:4]: You can use barrel export in pages.

**Review by jeetsdev — APPROVED** [review]
Everything looks good.
Just use barrel export to make the code cleaner.

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
```
Comment (govindappaarun) [inline public/manifest.json:2]: suggestion: rename the defaults in this file as well

File: src/components/TextFieldComp.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { FormControl, TextField } from '@mui/material';
+import { Box } from '@mui/system';
+import React from 'react';
+
+const TextFieldComp = () => {
+  const handleChange = () => {};
```
Comment (govindappaarun) [inline src/components/TextFieldComp.jsx:6]: suggestion: you keep consistency with the names, to just TextField, 
I understand why you have named, you can rename the one you are importing

File: src/components/TextFieldComp.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import { FormControl, TextField } from '@mui/material';
```
Comment (govindappaarun) [inline src/components/TextFieldComp.jsx:1]: suggestion: rename as TextField as TextInput 
rename the file name also to match TextField

**Review by govindappaarun — APPROVED** [review]
Looks good, 
Your code is nice and simple to read.

**Pull request merged**